opentelekomcloud/diff_suppress_funcs.go

```diff
@@ -139,3 +139,10 @@ func suppressLBWhitelistDiffs(k, old, new string, d *schema.ResourceData) bool {
 
 	return reflect.DeepEqual(old_array, new_array)
 }
+
+func suppressSchemaDiff(k, old, new string, d *schema.ResourceData) bool {
+	if strings.ToLower(old) == strings.ToLower(new) {
```

Comment:
The problem in #664 was difference between `v1.15.11-r1` and `v1.15.11`, not a case difference